components/NoteCard.tsx

```diff
@@ -0,0 +1,21 @@
+import { View, Text } from "react-native";
+
+export interface NoteCardProps {
+    noteTitle: string;
+    noteBody: string[];
+    bgColor: string;
+    textColor: string;
+}
+
+function NoteCard(props: NoteCardProps) {
```

Comment:
let's go ahead and deconstruct the `NoteCardProps` here, so we don't have to keep doing `props...` for each property! also, i think not deconstructing them may affect loading the props (potential NativeWind issue) - i was trying this out on my branch and the bgColor wouldn't show up until i deconstructed

Reply:
I'm not sure if I did this the right way (repeating myself a lot). Can you check is [this](https://github.com/anyaparekh/ShareAThought/blob/3183dac0f09fae900ba3d9b8c185ff49f79c67f9/components/NoteCard.tsx#L10) is right?

Reply:
Also worth noting I'm still facing a weird issue with styling loading even after expanding the props - not sure exactly what causes it. Sometimes I can get it to look as I want, sometimes I can't.

Reply:
I found the problem: you can't just specify the hex code because tailwind runs as a compile step and isn't dynamic, so you can't interpolate strings and runtime. My current solution is to change the props to take in a full tailwind string (eg.`bg-[#71816D]`) and have a map of strings to the tailwind strings (eg. `background: bg-[#71816D]`). Is there a better way to do this?

Reply:
great catch, you are totally correct! using complete class names seems like the best option to avoid this issue, so this looks good to me!